doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -0,0 +1,152 @@
+Introduction
+============
+
+Overview
+--------
+
+This Reference Architecture is focussed on OpenStack as the Virtualised
+Infrastructure Manager (VIM) chosen based on the criteria laid out in
+the `Reference Model <../../../ref_model/chapters/chapter01.md>`__.

```

Comment:
You do grep all md files and then do replace them to the right rst.
You can easily double check the overall work by running linkcheck after removing all markdown files.